frameworks/youth-risk-assessment/questions/care-information.yml

```diff
@@ -0,0 +1,45 @@
+type: checkbox
+question: Is there any information that will help with caring for this young person during the move?
+description: Care information
+hint: Select all that apply
+prefill: true
+options:
+  -
+    label: Trigger points
+    value: Trigger points
+    hint: For example, topics of discussion that should be avoided.
+    followup_comment:
+      label: Give details
+      validations:
+        -
+          type: required
+          message: Enter details
```

Comment:
I wonder if we should take the approach used in the PER here...and make the error messages more explicit? Otherwise if there are multiple the error summary will just read:

```
Enter details
Enter details
Enter details
```